cli/src/athenapdf.js

```diff
@@ -34,6 +39,7 @@ athena
     .option("-S, --stdout", "write conversion to stdout")
     .option("-A, --aggressive", "aggressive mode / runs dom-distiller")
     .option("-B, --bypass", "bypasses paywalls on digital publications (experimental feature)")
+    .option("-R, --request-header <key-value pair>", "Request header to send in the format 'key=value'. Both 'key' and 'value' are expected to be urlencoded", collect, [])
```

Comment:
I'm think if we should call it `--header` as that's what cURL calls it... but at the same time it might be confusing since people might think it's a PDF header. What do you think?